coderd/presets_test.go

```diff
@@ -0,0 +1,56 @@
+package coderd_test
+
+import (
+	"context"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+
+	"github.com/coder/coder/v2/coderd/coderdtest"
+	"github.com/coder/coder/v2/coderd/database"
+	"github.com/coder/coder/v2/coderd/database/dbauthz"
+	"github.com/coder/coder/v2/coderd/httpmw"
+	"github.com/coder/coder/v2/coderd/rbac"
+)
+
+func TestTemplateVersionPresets(t *testing.T) {
+	t.Parallel()
+
+	ctx := context.Background()
```

Comment:
Rather use `testutil.Context` with a timeout to prevent this test from never stopping.

Reply:
done.